lib/src/map/built_map.dart

```diff
@@ -61,10 +61,6 @@ abstract class BuiltMap<K, V> {
   ///
   /// `K` and `V` must not be `dynamic`.
```

Comment:
How about:

`K` and `V` are inferred from `map`, and must not be `dynamic`.

?

Reply:
Sounds good, corrected! :)